src/components/sections/robots/robot.tsx

```diff
@@ -0,0 +1,182 @@
+
+export default function SponsorImages2023() {
+  const logoFolder = "https://cdn.team2658.org/web-public/robots"; // last character CANNOT be a '/'
+	const robots = [
+		{
+			name: "Charged Up",
+			year: "2022",
+			path: `${logoFolder}/ChargedUp.jpeg`,
+			width: 300,
+			height: 300,
+			description: ``,
+			pos: 'center'
+		},
+		{
+            name: "Rapid React",
+			year: "2021",
+			path: `${logoFolder}/RapidReact.png`,
+			width: 300,
+			height: 300,
+			description: ``,
+			pos: 'center'
+		},
+		{
+			name: "Infinite Recharge",
+			year: "2020",
+			path: `${logoFolder}/InfiniteRecharge.jpeg`,
+			width: 300,
+			height: 300,
+			description: ``,
+			pos: 'center'
+		},
+		{
+			name: "Destination: Deep Space",
+			year: "2019",
+			path: `${logoFolder}/DeepSpace.png`,
+			width: 300,
+			height: 300,
+			description: ``,
+			pos: 'center'
+		},
+		{
+			name: "Power Up",
+			year: "2018",
+			path: `${logoFolder}/PowerUp.jpeg`,
+			width: 300,
+			height: 300,
+			description: ``,
+			pos: 'center'
+		},
+		{
+			name: "Steamworks",
+			year: "2017",
+			path: `${logoFolder}/Steamworks.jpeg`,
+			width: 300,
+			height: 300,
+			description: ``,
+			pos: 'center'
+		},
+		{
+			name: "Stronghold",
+			year: "2016",
+			path: `${logoFolder}/Stronghold.jpeg`,
+			width: 300,
+			height: 300,
+			description: ``,
+			pos: 'center'
+		},
+		{
+			name: "Recycle Rush",
+			year: "2015",
+			path: `${logoFolder}/RecycleRush.jpeg`,
+			width: 300,
+			height: 300,
+			description: ``,
+			pos: 'center'
+		},
+		{
+			name: "Aerial Assist",
+			year: "2014",
+			path: `${logoFolder}/AerialAssist.jpeg`,
+			width: 300,
+			height: 300,
+			description: ``,
+			pos: 'center'
+		},
+		{
+			name: "Ultimate Ascent",
+			year: "2013",
+			path: `${logoFolder}/UltimateAscent.jpeg`,
+			width: 300,
+			height: 300,
+			description: ``,
+			pos: 'center'
+		},
+		{
+			name: "Rebound Rumble",
+			year: "2012",
+			path: `${logoFolder}/ReboundRumble.jpeg`,
+			width: 300,
+			height: 300,
+			description: ``,
+			pos: 'center'
+		},
+		{
+			name: "Logo Motion",
+			year: "2011",
+			path: `${logoFolder}/LogoMotion.jpeg`,
+			width: 300,
+			height: 300,
+			description: ``,
+			pos: 'bottom'
+		},
+		{
+			name: "Breakaway",
+			year: "2010",
+			path: `${logoFolder}/Breakaway.jpeg`,
+			width: 300,
+			height: 300,
+			description: ``,
+			pos: 'center'
+		},
+		{
+			name: "Lunacy",
+			year: "2009",
+			path: `${logoFolder}/Lunacy.jpeg`,
+			width: 300,
+			height: 300,
+			description: ``,
+			pos: 'center'
+		},
+		{
+			name: "Overdrive",
+			year: "2008",
+			path: `${logoFolder}/Overdrive.png`,
+			width: 300,
+			height: 300,
+			description: ``,
+			pos: 'bottom'
+		},
+	];
+
+	return (
+		<div
+			className="mx-[5vw] my-[160px] rounded-xl bg-black p-8"
+			id="robots"
+		>
+			{robots.map((item) => (
+			<div key={"robot-" + item.name} className="grid grid-cols-1 gap-10 md:grid-cols-1 xl:grid-cols-1 mx-[10vw]">

```

Comment:
for ``key={"robot-" + item.name}``, make sure the ``item.name`` doesn't have spaces, camel case them instead. could use javascript string formatting. Alternatively, instead of changing ``item.name``, you could implement this by using:
```typescript
item.name.split(" ").join("");
```